scripts/trainer_extract_args.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+import os
+import re
+import sys
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def get_model_name():
+    text = os.environ.get("PR_DESCRIPTION")
+
+    match = re.search(r"Train on Taskcluster:\s+([a-z_1-9]+)", text)
+
+    if not match:
+        logger.error("There is no match found for keyword 'Train on Taskcluster:'")
+        sys.exit(1)
```

Comment:
We should not do the flow control from this function. It is better to do it from the main:
```suggestion
        return None
```